tests/core/test_basics.py

```diff
@@ -17,11 +20,11 @@
 async def test_request(endpoint: Endpoint) -> None:
     endpoint.subscribe(
         DummyRequestPair,
-        lambda ev: endpoint.broadcast(
+        lambda ev: cast(None, asyncio.ensure_future(endpoint.broadcast(
```

Comment:
I really think we should add a `broadcast_nowait` that does this because:

- the `asyncio.ensure_future` becomes repetitive and even more involved if it also needs a `cast`
- in a `broadcast_nowait` we can make sure that `asyncio_ensure_future` puts the `broadcast` call on `loop=self._loop` which might be important for situations where the endpoint does not run on the default loop.

Reply:
Okay, I'm not opposed to adding a `broadcast_nowait` :+1: 

> in a broadcast_nowait we can make sure that asyncio_ensure_future puts the broadcast call on loop=self._loop which might be important for situations where the endpoint does not run on the default loop.

What's a situation where you would have two loops in the same process? That sounds dangerous!

Reply:
I found it weird and scary, too when I first encountered it but we even had this within Trinity before the JSON-RPC server moved into an isolated plugin. Here is a link to [old Trinity code that did this](https://github.com/ethereum/trinity/blob/7b1fec6249652df01535ac900c190b0b538d2830/trinity/nodes/base.py#L143-L154).

There's also a comment on it that tries to explain the rational. I'm happy we got rid of it but there may be valid reasons for people to use that pattern for certain scenarios.